### DISCUSSION TITLE
beta 42

### DISCUSSION THREAD

**Discussion by MichelMoser** [vdemichev/DiaNN#1023] [Q&A] [answered]
Dear @vdemichev , 

I saw the new plotter.exe in beta 42. 
Will you be releasing any documentation on how to use it? 

I am working on XIC visualization in R as well but you might be well ahead :) 

Thank you, 
Michel

**Comment by vdemichev** [comment, marked as answer]
Hi Michel,

The plotter is the old one, has not been updated.

About XICs, please try passing the --xic option to DIA-NN. Then once the analysis is finished, can press the 'Viewer' button. Also, can access those XICs with R 'arrow' package (read_parquet()), XICs are saved in a subfolder created in the output folder.

Best,
Vadim
